includes/class-used-css.php

```
@@ -0,0 +1,1206 @@
+			if ( ! $fs ) {
+				return false;
+			}
+
+			if ( ! Util::prepare_cache_dir( $dir_path ) ) {
+				return false;
+			}
+
+			// Atomic write: write to a temporary file, then rename to prevent
+			// concurrent requests from producing truncated/interleaved output.
+			$tmp_path = $file_path . '.tmp.' . wp_rand();
+			$written  = $fs->put_contents( $tmp_path, $css, FS_CHMOD_FILE );
```

PHPStan (2.2.6)

[warning] Constant FS_CHMOD_FILE not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)